########################################
# Messages
########################################
string start
string end
string path_id
float64[] cx
float64[] cy
float64[] cyaw

